posthog/tasks/email.py

```diff
@@ -97,6 +97,24 @@ def send_email_verification(user_id: int, token: str) -> None:
     message.send()
 
 
+@app.task(max_retries=1)
+def send_over_quota_but_not_dropped_email_to_cs(organization_id: int) -> None:
+    organization: Organization = Organization.objects.get(pk=organization_id)
+    message = EmailMessage(
+        campaign_key=f"over_quota_but_not_dropped-{organization_id}-{timezone.now().timestamp()}",
```

Comment:
Could we perhaps use the campaign key to achieve the caching? If we make the key use the calendar week like `now().isocalendar().week` mixed with the year, then we will only send one message per week which achieves the caching mechanism _and_ makes it easier to debug as we can easily inspect and modify postgres, whereas redis is a bit more transient.

Or alternatively extend the MessagingRecord to support multiple emails being sent for the same campaign_key with some sort of ttl. Like 

```
message.send_again_after =  timedelta(days=7)
if message.send_again_after and last_message.sent_at <  now() - message.send_again_after:
```

Reply:
> If we make the key use the calendar week like now().isocalendar().week mixed with the year, then we will only send one message per week

Kind of... if a message is sent the last day of week 10, then another one will be sent the next day since it's week 11. So I don't think this is the right solution.

> Or alternatively extend the MessagingRecord to support multiple emails being sent for the same campaign_key with some sort of ttl.

I can see other situations where we want to resend emails and have a record of the same exact email being sent on different days. I'll extend it in this way.